README.md

```diff
@@ -6,13 +6,22 @@ This repository shows various ways of deploying a vision model (TensorFlow) from
 
 ## Methods covered
 
-- [x] TensorFlow Serving locally ([blog post from Hugging Face](https://huggingface.co/blog/tf-serving-vision))
-- [x] Vertex AI (without TensorFlow Runtime)
-- [ ] Vertex AI (with TensorFlow Runtime)
-- [x] Kubernetes with GKE
-
-Know more about the optimized TensorFlow runtime [here](https://cloud.google.com/vertex-ai/docs/predictions/optimized-tensorflow-runtime).
-
-A more detailed README will be added later. 
-
+- [x] [Local TensorFlow Serving](https://github.com/sayakpaul/deploy-hf-tf-vision-models/blob/main/hf_vision_model_tfserving.ipynb) | [Blog post from 🤗](https://huggingface.co/blog/tf-serving-vision)
+  - We cover how to locally deploy a Vision Transformer (ViT) model from 🤗 Transformers with TensorFlow Serving. 
+  - With this, you will be able to serve your own machine learning models in a standalone Python application.
+- [x] [TensorFlow Serving on Kubernetes(GKE)](https://github.com/sayakpaul/deploy-hf-tf-vision-models/tree/main/hf_vision_model_tfserving_gke) | [Blog post from 🤗](https://huggingface.co/blog/deploy-tfserving-kubernetes)
```

Comment:
Since this one has a `locust` component too, should we move the current `locust` directory inside `hf_vision_model_tfserving_gke` and call it out from this section? 

WDYT?

Reply:
That makes sense!